GoogleForCreators/web-stories-wp
# Code Quality: Remove showSemanticHeadings feature flag code

## Context

This feature has been enabled by default in v1.23.0.

Feature PR https://github.com/GoogleForCreators/web-stories-wp/pull/11888



## Summary

Removes showSemanticHeadings feature flag



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1) Navigate to a Story within the Editor.
2) Add multiple text elements with variying font sizes.
3) Note that the feature is on by default ... You should be able to see in the Style section of the sidebar, underneath a new Accessibility tab, the type of semantic heading.


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #11996
